doc/lua_api.txt

```diff
@@ -4864,20 +4862,22 @@ These functions return the leftover itemstack.
     * returns `itemstack, success`
 * `minetest.item_place_object(itemstack, placer, pointed_thing)`
     * Place item as-is
-* `minetest.item_place(itemstack, placer, pointed_thing, param2)`
-    * Use one of the above based on what the item is.
+    * returns the leftover itemstack
+    * **Note**: This is never called and exists solely for compatibility.
```

Comment:
How about: `**Note**: This function is deprecated.`
`never called` is not so accurate, and "deprecated" would kind of indicate that it's only there for compatibility reasons.

Reply:
"Deprecated" however does not imply that the function is never called